Adding initial Framework API Documentation

MMetelko opened the pull request:
Please review and indicate if any refinements to the look of these API documents is desired before publishing to main.  This is an initial submission, no effort has been made yet to make it look the same across repositories.

MMetelko committed:
Saving current updates, outlines, not complete

MMetelko committed:
Content editing.

MMetelko committed:
Add Framework APIs for riaps-pycom

MMetelko committed:
Add riaps-core API doc and a blank riaps-timesync API doc (placeholder for now)

MMetelko committed:
Timesync API is currently a placeholder

MMetelko committed:
Adding timesync API doc

gkarsai commented:
Two things:
- On the main intro pages (with the 3 subsections) indicate that this is work in progress
- The RIAPS_Core section needs some content on its landing page (explainin what riaps_core is)

imadari committed:
doc update

MMetelko committed:
Update documentation for core and pycom

MMetelko commented:
Added requested information.

imadari committed:
doc update

MMetelko committed:
Update docprocess.md

gkarsai committed:
Merging in LF/OS related files.

MMetelko committed:
Removed API documentation and referenced the ReadTheDocs pages
Updated the tutorial information to simplify and make Hello World video the key information link
Started launch.md outline, not complete, plan to focus on commands and troubleshooting of the system when running these tools (information not covered in the hello world video).

MMetelko committed:
Update tutorials.md

MMetelko committed:
Update tutorial information
Launch tutorial still in work, but general concept outlined

MMetelko committed:
Idea logging (TODO)

gkarsai committed:
add CONTRIBUTING section

gkarsai committed:
minor edit

MMetelko committed:
More updates to launch.md
Placeholder for TOC, not sure the tag works in github

timkrentz committed:
Small wording changes

MMetelko committed:
Updates to the tutorials

timkrentz committed:
Add description, video link, and code

timkrentz committed:
Update hello-world.md

timkrentz committed:
HTML used for youtube embedding

From: https://github.com/github/pages-gem/issues/609

timkrentz committed:
Testing Youtube embedding

timkrentz committed:
Try fixing jekyll include

MMetelko committed:
Updated tutorials for consistency
Added Page Content section

MMetelko committed:
Set section levels like others in this repository

MMetelko committed:
Fixed a few links

MMetelko committed:
Merge branch 'master' into develop

MMetelko commented:
Tutorial updates have been made for consistency and have page content section at top
First Video Tutorial has been added, others are still placeholders

gkarsai committed:
Adding CONTRIBUTING section.

gkarsai committed:
Undo previous, already done.
